tests/test_browser.py

```diff
@@ -44,7 +44,8 @@ def do_GET(s):
         s.sendheaders([],length)
         s.wfile.write(data[start:end+1])
 
-  expectedConns = 11
+  # CORS preflight makes OPTIONS requests which we need to account for.
+  expectedConns = 22
```

Comment:
hmm, could this just be 100 or something like that? or is there a benefit to making it precisely 22?

Reply:
22 is the smallest value that works.  There's no reason it couldn't be something bigger, though.

Reply:
The previous value was the smallest value that worked AFAICT, so that's why I didn't go with anything larger.

Reply:
Fair enough. I don't know if this matters, I was just curious if there was something more that you noticed  there.